data_driven_onboarding/schema.py

```diff
@@ -73,9 +73,16 @@ class DDOSuggestionsResult(graphene.ObjectType):
     )
 
     hpd_open_violation_count = graphene.Int(
+        required=True,
```

Comment:
(This line wasn't in your original PR, I added it in c41122fd78fb65a247672525b2615734e04c83ac).  Because this can no longer ever be NULL, I've updated the GraphQL schema (the machine-readable representation of how our API works) to indicate that the field will never be null too.